pkg/util/github/github.go

```diff
@@ -190,7 +190,16 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	binaryPath := ""
 	switch {
 	case strings.HasSuffix(assetTmpFile, TAR):
-		tarFile := assetTmpFile[:len(assetTmpFile)-3]
+		tarFile := ""
+		if osType == minishiftos.LINUX {
+			// https://github.com/openshift/origin/issues/12663
```

Comment:
What if https://github.com/openshift/origin/issues/12576 gets "fixed" and the filename contains '-' again, then we have to adjust the code again. This will result in versions where our code will break and versions where it will work.